go/interpreter/lfvm/instructions_test.go

```diff
@@ -1146,3 +1146,103 @@ func TestGenericCreate_MaxInitCodeSizeIsNotCheckedBeforeShanghai(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckSizeOFfsetUintOverflow_ReturnsAsExpected(t *testing.T) {
```

Comment:
I think we are missing a case:
`offset.Uint64()+size.Uint64() < offset.Uint64() `
Are we?